rnd/autogpt_libs/autogpt_libs/auth/jwt_utils.py

```diff
@@ -0,0 +1,20 @@
+import jwt
+from typing import Dict, Any
+from .config import settings
+
+
+def parse_jwt_token(token: str) -> Dict[str, Any]:
```

Comment:
Is it possible to be more specific about the type the token is?

Reply:
can you explain what you mean, please